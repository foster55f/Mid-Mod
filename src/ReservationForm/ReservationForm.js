import React, { Component } from 'react';


class ReservationForm extends Component{
    constructor() {
        super();
        this.state = {
            id: '',
            name: '',
            date: '',
            time: '',
            number:''
        }
    }
}

export default ReservationForm