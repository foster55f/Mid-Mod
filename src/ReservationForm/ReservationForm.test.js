import React from 'react';
import { shallow } from 'enzyme'
import ReservationForm from './ReservationForm'
import { isTSAnyKeyword } from '@babel/types';

describe('ReservationForm', () => {
    let wrapper;
    const mockAddReservation = jest.fn()

    beforeEach(() => {
        wrapper=shallow(<ReservationForm addReservation={mockAddReservation}/>)
    })

    it('should match the snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
    
})