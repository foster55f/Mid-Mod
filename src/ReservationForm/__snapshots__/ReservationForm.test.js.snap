// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationForm should match the snapshot 1`] = `
<form>
  <input
    name="name"
    onChange={[Function]}
    placeholder="Name"
    type="text"
    value=""
  />
  <input
    name="number"
    onChange={[Function]}
    placeholder="RoomNumber"
    type="number"
    value=""
  />
  <input
    max="12-31-2019"
    min="01-01-2019"
    name="date"
    onChange={[Function]}
    type="date"
    value=""
  />
  <button
    onClick={[Function]}
  >
    Make Reservation
  </button>
</form>
`;
